content/en/_index.html

```diff
@@ -13,247 +13,1763 @@
+    <div class="flex gap-md margin-lg-top text-centered-mobile" id="main-bt">
       <a href="https://notaryproject.dev/docs/quickstart/">
-        <button class="button is-medium is-dark get-started">Get started</button>
+        <button class="gs-bt">Get started</button>
       </a>
-      <a href="https://killercoda.com/notaryproject/scenario/notation" target="_blank" rel="noopener noreferrer">
-        <button class="button is-medium is-dark demo">Try it
-          <span></span>
-          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="white" style="height: 1rem;">
-          <path fill-rule="evenodd" d="M4.25 5.5a.75.75 0 00-.75.75v8.5c0 .414.336.75.75.75h8.5a.75.75 0 00.75-.75v-4a.75.75 0 011.5 0v4A2.25 2.25 0 0112.75 17h-8.5A2.25 2.25 0 012 14.75v-8.5A2.25 2.25 0 014.25 4h5a.75.75 0 010 1.5h-5z" clip-rule="evenodd" />
-          <path fill-rule="evenodd" d="M6.194 12.753a.75.75 0 001.06.053L16.5 4.44v2.81a.75.75 0 001.5 0v-4.5a.75.75 0 00-.75-.75h-4.5a.75.75 0 000 1.5h2.553l-9.056 8.194a.75.75 0 00-.053 1.06z" clip-rule="evenodd" />
-        </svg>
-        </button>
+      <a href="https://github.com/notaryproject/notation">
+        <button class="gt-bt">GitHub</button>
       </a>
     </div>
   </div>
-  <div class="img-long is-hidden-mobile">
-    <iframe src="https://www.youtube.com/embed/ZMNUazoGwdk" frameborder="0"></iframe>
+  <div class="img-long is-hidden-mobile" id="yt">
+    <iframe
+      src="https://www.youtube.com/embed/ZMNUazoGwdk"
+      frameborder="0"
+      id="yt-video"
+    ></iframe>
   </div>
 </section>
-<section>
-  <p><b>Contributed by the community, in collaboration with</b></p>
-  <div class="width-80 is-centered space-between is-align-items-center is-mobile-responsive text-centered-mobile flex">
-    <div class="img-container-smd is-centered">
-      <img src="aws_logo.svg" alt="AWS logo" />
-    </div>
-    <div class="img-container-mdl is-centered">
-      <img src="cncf_logo.svg" alt="CNCF logo" />
-    </div>
-    <div class="img-container-sm is-centered">
-      <img src="docker_logo.svg" alt="Docker logo" />
-    </div>
-    <div class="img-container-sm is-centered">
-      <img src="microsoft_logo.svg" alt="Microsoft logo" />
-    </div>
-  </div>
-  <hr class="broken-line" />
-</section>
-{{< /blocks/cover >}}
-
-<section class="bg-container text-centered">
-  <div class="text-container">
-    <h1 class="title">Scenarios we fit and problems we solve for</h1>
-    <p class="width-60 is-centered font-size-md">
-      Signing and validating software artifacts, ensure they have not been tampered with and provide security policies
-      to determine which validated artifacts are allowed to be used in your systems
+
+<!-- {{< /blocks/cover >}} -->
+<div class="wf-block">
+  <div class="sn-cont">
+    <h1 class="sn-hd">Scenarios we fit and problems we solve for</h1>
+    <p class="sn-desc">
+      Sign and validate software artifacts to ensure integrity, prevent
```

Comment:
Sign and validate software artifacts, and ensure they have not been tampered with. Provide security policies to determine which validated artifacts are allowed to be used in your systems